src/main/java/org/candlepin/pki/certs/SCACertificateGenerator.java

```diff
@@ -296,7 +296,17 @@ private Content createContent(Owner owner, Environment environment) {
             path += "/" + Util.encodeUrl(owner.getKey());
         }
         if (environment != null) {
-            path += "/" + Util.encodeUrl(environment.getName());
+            StringBuilder envPrefix = new StringBuilder();
+            for (String chunk : environment.getName().split("/")) {
+                if (chunk.isEmpty()) {
+                    continue;
+                }
+
+                envPrefix.append("/");
+                envPrefix.append(Util.encodeUrl(chunk));
+            }
```

Comment:
This is the logic that was removed from the following:
https://github.com/candlepin/candlepin/pull/4333/files#diff-6328ae639f7b8abb7bb741f14992daffb9f98d6a73d7fd60f3b6d0aac71e343fL89-L97